Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_First_SSO_Access.yml

```diff
@@ -0,0 +1,1906 @@
+    message:
+      to:
+        simple: Analyst
+      subject:
+        simple: Based on the investigation details, is this a true positive?
+      body:
+      methods:
+      - email
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Based on the investigation details, is this a true positive?
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: multiSelect
+        options: []
+        optionsarg:
+        - simple: "Yes"
+        - simple: "No"
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: Based on the investigation details, is this a true positive?
+      description: ""
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "157":
+    id: "157"
+    taskid: b67ff8da-2897-42a3-8e54-4b74fe65048b
+    type: playbook
+    task:
+      id: b67ff8da-2897-42a3-8e54-4b74fe65048b
+      version: -1
+      name: Cortex XDR - First SSO Access - Set Verdict
+      description: |-
+        This playbook determines the alert’s verdict based on the results of multiple checks.
+        By default, if at least two of the checks' results are true, the verdict is set to malicious.
+        else if only one check's results are true, the verdict is set to suspicious.
+        If none of the conditions is true,  the verdict is set to non-malicious.
+        It is possible to change the threshold value of the inputs to change the sensitivity of the verdict
+      playbookName: Cortex XDR - First SSO Access - Set Verdict
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "34"
+    scriptarguments:
+      AlertName:
+        complex:
+          root: inputs.AlertName
+      FailedlogonFromASNThreshold:
+        complex:
+          root: inputs.FailedlogonFromASNThreshold
+      FailedlogonUserThreshold:
+        complex:
+          root: inputs.FailedlogonUserThreshold
+      MaliciousVerdictThreshold:
+        simple: "2"
+      NumOfFailedLogon:
+        complex:
+          root: NumOfFailedLogon
+      NumOfFailedLogonASN:
+        complex:
+          root: NumOfFailedLogonASN
+      NumOfOktaSuspiciousActivities:
+        complex:
+          root: NumOfOktaSuspiciousActivities
+      NumOfOktaSuspiciousUserAgent:
+        complex:
+          root: NumOfOktaSuspiciousUserAgent
+      NumOfXDRAlerts:
+        complex:
+          root: ArraySize
+      PermanentCountry:
+        complex:
+          root: PermanentCountry
+      RelatedCampaign:
+        complex:
+          root: RelatedCampaign
+      SuspiciousVerdictThreshold:
```

Comment:
Should be populated the main playbook as well . also, need to specify it should be lower than the "MaliciousVerdictThreshold" input.

Reply:
Done.